docs/content/gs-adding-users-proc.adoc

```diff
@@ -1,19 +1,23 @@
 [id='gs-adding-users-proc']
 
-= User Management
-
-This section covers how to manage users within the Integreatly environment.
-
-== Accessing the SSO Admin Console
-- Login to the OpenShift console as the `customer-admin` user (The credentials for this user were sent with your welcome email).
-- Navigate to the `sso` project.
-- Go to *Applications* > *Routes*
-- Click on the `sso` route hostname to open the Single Sign-On (SSO) landing screen.
-
-image:../assets/images/gs-adding-users-sso-route.png[SSO route]
-
-- Click on the `Administration Console` link on the SSO landing screen.
-- Login to the SSO administration console as the `customer-admin` user.
+= Managing Users
+
+How to manage users within the {PRODUCT} environment.
+
+== Accessing the Red Hat Single Sign-On administration console
+. Login to the OpenShift console as the `customer-admin` user. 
+. Navigate to the *sso* project.
+. Retrieve the credentials for the Red Hat Single Sign-On administration console:
```

Comment:
@JameelB  isn't this necessary?

Reply:
This shouldn't be needed as they will be sent the customer-admin credentials in the welcome email. They can then use these to login to the rh-sso openshift realm

Reply:
@maleck13  where is this mail been written? cos we should  add the url for the RHSSO there, right?